includes/class-wcpdf-assets.php

```diff
@@ -148,6 +148,19 @@ public function backend_scripts_styles ( $hook ) {
 						'disable_free',
 						'use_latest_settings',
 					) ),
+					'pointers'                  => array(
+						'wcpdf_document_settings_sections' => array(
+							'target'        => '.wcpdf_document_settings_sections',
+							'content'       => sprintf( '<h3>%s</h3><p>%s</p>', __( 'Document settings', 'woocommerce-pdf-invoices-packing-slips'), __( 'Select a document in the dropdown menu above to edit its settings.', 'woocommerce-pdf-invoices-packing-slips') ),
```

Comment:
WPCS

Reply:
@alexmigf Are you referring here to the `sprintf()` set on one line?

Reply:
Yes, here `'woocommerce-pdf-invoices-packing-slips')` => `'woocommerce-pdf-invoices-packing-slips' )`, 2 times.

Reply:
@Terminator-Barbapapa it's the missing space after the text domains.

Reply:
Fixed here: 4cbd187db9404e3b01f33f3c61150f333e44b56a